README.md

```diff
@@ -67,11 +67,23 @@ app.register(fastifySwagger, {
     servers: [],
   },
   transform: jsonSchemaTransform,
+
   // You can also create transform with custom skiplist of endpoints that should not be included in the specification:
   //
   // transform: createJsonSchemaTransform({
   //   skipList: [ '/documentation/static/*' ]
   // })
+
+  // In order to create refs to the schemas, you need to provide the schemas to the transformObject using createJsonSchemaTransformObject
+  //
+  // transformObject: createJsonSchemaTransformObject({
```

Comment:
can you provide a full example (can be in a separate section), which would demonstrate what kind of endpoint definition would match with what kind of schema? current example won't actually work, there is no User entity used in app registered.